src/Generation/UnitTestsGenerator.php

```diff
@@ -298,7 +298,7 @@ private function testExceptionalCaseNormal(MethodDetails $method): PhpMethod
                 $requestPerField->map(fn ($x) => $x->initCode),
                 AST::try(
                     $client->instanceCall(AST::method($method->methodName))($requestCallArgs),
-                    '// If the $client method call did not throw, fail the test',
+                    '// If the $gapicClient method call did not throw, fail the test',
```

Comment:
Nit: This is kind of a hack, but you could put `$client->toCode()` here and it would emit `$gapicClient`, so that we don't have a hard coded value? Or perhaps even try just putting the `$client` var in there and see if it generates it properly (without you having to call `toCode()` directly). Alternatively, just use a variable for the AST::var name and for here.

Kind of applies elsewhere. lmk wyt

Reply:
I'll update to use a variable, good call out.

Reply:
We could probably do some very similar changes for other fields to help improve maintainability, let's reserve that for future updates.

Reply:
Yeah good point. Thank you!